lib/OBSWebSocket.js

```diff
@@ -37,20 +35,20 @@ class OBSWebSocket extends Socket {
       const messageId = args['message-id'] = generateMessageId(); // eslint-disable-line no-multi-assign
 
       // Submit the request to the websocket.
-      log.debug('[Core:send]', messageId, requestType, args);
+      debug('[send] %s %s %O', messageId, requestType, args);
       this._socket.send(JSON.stringify(args));
```

Comment:
Opened an issue for later.
https://github.com/haganbmj/obs-websocket-js/issues/34

Pretty sure it's as easy as just rejecting early if there's no socket connection.
The plugin itself will return a reject if you're not authenticated, so that should be sufficient.